docs/userGuide/syntax/includes.md

```diff
@@ -56,6 +56,15 @@ When setting the `id` of a fragment, be careful not to clash with heading anchor
 </box>
 
 <include src="panels.md#script_and_styles_warning"></include>
+<div id="baseUrl-warning">
+<box type="warning" header="Add `{{ '{{ baseUrl }}' }}` to local URLs to always point to the same target!">
+    
+For links within `<include>` to always point to the same target, make it an absolute link with `{{ '{{ baseUrl }}' }}` included.  
```

Comment:
Could you include a negative example so users know what what not to do as well?

Reply:
yep sure included! lmk if its not clear or anything